Library/Library/LibraryList.aspx.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+
+namespace Library
+{
+    public partial class LibraryList : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            if (!IsPostBack)
+            {
+                DataTable dt = DatabaseHelper.Retrieve(@"
+                    SELECT [Library].LocationName
+                    FROM [Library]
```

Comment:
It's a good idea for list queries to include an `ORDER BY` clause. Without an explicit sort, you're not guaranteed the same order of results.